Makefile

```
@@ -7,16 +7,21 @@ TESTTARGETS := $(shell go list -e ./... | egrep -v "/(vendor)/")
 # ex, -v
 TESTOPTS := 
 
+.PHONY: default
+default: build
+
+.PHONY: all
+all: check build test
+
 .PHONY: check
 check: ## Lint code
 	gofmt -s -l $(shell go list -f '{{ .Dir }}' ./... ) | grep ".*\.go"; if [ "$$?" = "0" ]; then gofmt -s -d $(shell go list -f '{{ .Dir }}' ./... ); exit 1; fi
 	go vet ./cmd/... ./pkg/...
 
 .PHONY: build
-build: test ## Build binary
+build: check ## Build binary
```

```suggestion
build: test check ## Build binary
```

We probably want to test before `build`?